### PR TITLE
Draft PR for recording verification ops in VCD

### PR DESCRIPTION
**Pull request by chick**

### CODE REVIEW

**Comment by chick** [comment]
@ekiwi Updated the tests so they actually look at the VCD output.
Just for fun, I implemented the cover statement to add the cover counts into the VCD file, but I don't
have a conviction that it's the right way to go

**Comment by chick** [comment]
Addresses Issue #347

File: src/test/scala/treadle/VerificationSpec.scala [hunk 1]
```diff
@@ -125,5 +127,110 @@ class VerificationSpec extends AnyFreeSpec with Matchers {
       }
       output.toString should include("input was not less that 0x7f")
     }
+
+    "asserts from verification should go high when triggered and be visible" in {
+      val targetDir = s"test_run_dir/verify_assumes_should_log_to_vcd"
```
Comment (ekiwi) [inline src/test/scala/treadle/VerificationSpec.scala:132]: this should be `asserts` not `assumes`

File: src/test/scala/treadle/VerificationSpec.scala [hunk 2]
```diff
@@ -125,5 +127,110 @@ class VerificationSpec extends AnyFreeSpec with Matchers {
...
+          |
+          |  module test:
+          |    input clock : Clock
+          |    input reset : AsyncReset
+          |
+          |    inst c of child
+          |    c.clock <= clock
+          |    c.reset <= reset
+          |
+          |    cover(clock, lt(c.count, UInt(5)), UInt(1), "") : leq_assert
```
Comment (ekiwi) [inline src/test/scala/treadle/VerificationSpec.scala:205]: Maybe name this `leq_cover` instead of `leq_assert`

**Review by ekiwi — APPROVED** [review]
I think this is good for now. I would prefer for the assert signal to go high one cycle earlier, but we can try to address that in a followup PR. This PR already improves upon the status-quo.

Please address the two small comments and then feel free to merge.

**Pull request merged**